build.gradle

```diff
@@ -27,7 +27,6 @@ project.version = "1.42.0" // {x-version-update:gax:current}
 
 ext {
   // When upgrading grpc, make sure to upgrade opencensusVersion to be consistent with grpc.
-  grpcVersion = '1.18.0'
   opencensusVersion = '0.18.0'
```

Comment:
Should opencensus be updated to 0.19.2 in line with https://github.com/grpc/grpc-java/commit/b03a793405bf30b5f5c524603b6ac81ebaece3d4#diff-c197962302397baf3a4cc36463dce5ea?

Also, can you move the above comment `// When upgrading grpc...` to the place that you've introduced the grpc bom?

Reply:
Ahh good catch!

Instead of moving stuff around, I decided to create the grpcVersion variable again. (This creates less of an overall diff.)